codespell_lib/data/dictionary.txt

```diff
@@ -34643,29 +34643,55 @@ refect->reflect
 refected->reflected
 refecting->reflecting
```

Comment:
Just noticed lots of preexisting issue here:

1. Actually `refect` is an English verb:
   * [Merriam-Webster](https://www.merriam-webster.com/dictionary/refect)
   
   If you don't mind, could you help moving these to _rare_?
   ```
   refect->reflect
   refected->reflected
   refecting->reflecting
   refects->reflects
   ```